### ISSUE TITLE
Auto-translations (or optional translation) for non-english sites?

### ISSUE THREAD

**Issue by rubyFeedback** [zedeus/nitter#436]
Hey there nitter devs,

I recently found out about it and twitter has been annoying me with pop-ups and what not (I stopped
using my account there after they kept on censoring my tweets arbitrarily).

I follow matz a lot:

https://nitter.net/i/status/1440289873294811141

But he tends to write in japanese most of the time.

I remember twitter had an auto-translation. This was not perfect but I could
at the least get an idea what matz was writing about.

Would it be possible to get some kind of auto-translations too? I don't know
if there are open source libraries (I know too little about that domain), but
if there are then perhaps one can query that API and translate as-is. Not
everyone needs this so perhaps it could be a small button or a separate
sub-"page" (like API or something "/english" as the trailing part of the
URI). It would be quite nice to have and perhaps others may want this
too.

If that is too much work to implement and such then please just disregard
completely. Feel free to close this issue at any moment in time. Thanks
for reading!

**Comment by gigigaz** [comment]
@zedeus Are there any plans to introduce auto-translation in the near future?

**Comment by zedeus** [comment]
No, it would be extremely taxing on instances with more than a handful of users. For people using Chromium-based browsers, they can use the browser's built-in auto-translate feature. A client-side feature may be added which allows clicking a button to translate a single tweet via JavaScript, but I consider it very low priority.
